<launch>

    <node pkg="move_base" type="move_base" respawn="false" name="move_base_node" output="screen">
        <rosparam file="$(find box_finder)/config/move_base_params.yaml" command="load" />
        <rosparam file="$(find box_finder)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find box_finder)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find box_finder)/config/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find box_finder)/config/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find box_finder)/config/base_global_planner_params.yaml" command="load" />
    </node>

</launch>
